modules/virt-enabling-disabling-vm-delete-protection-cli.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-cli_{context}"]
```

Comment:
IMHO these should be two separate one-command procedures, not a single one. Probably want to modularize these some time later prior to DITA migration.

Reply:
I can do that in another PR; thanks.